CMakeLists.txt

```diff
@@ -1244,9 +1003,224 @@ if(CONFIG_USERSPACE)
   )
 endif()
 
+if(CONFIG_USERSPACE OR CONFIG_HAS_DTS)
```

Comment:
Why isn't `gen_handles.py`  being executed inside this if statement?

Reply:
because the `gen_handles.py` is only dependent on `HAS_DTS`, not `USERSPACE`.